Pusher.Connections.Net/WebSocketConnection.cs

```diff
@@ -6,132 +6,136 @@
 

```

Comment:
In this file, spaces seem to have been replaced with tabs, making it very hard to see what has actually changed.